content/beta/asset-manager/swagger/resources/asset_families/definitions/asset_family.yaml

```diff
@@ -54,12 +54,75 @@ properties:
                 type: string
               mode:
                 type: string
-
+  transformations:
+    type: array
+    description: The transformations to perform on source files in order to generate new files into your asset attributes. To understand the format of this property, see <a href='/beta/asset-manager/transformations-focus.html'>here</a>.
+    x-validation-rules: To know more about the validation of this property, take a look at our transformation format.
+    default: {}
+    items:
+      type: object
+      required: ['label', 'target', 'source', 'operations']
+      properties:
+        label:
+          type: string
+          description: The name of the transformation
+        filename_suffix:
+          type: string
+          description: The suffix that will be append to the source filename to generate the target filename. More details <a href='/beta/asset-manager/transformations-focus.html#target-filename'>here</a>.
+        filename_preffix:
+          type: string
+          description: The prefix that will be prepend to the source filename to generate the target filename. More details <a href='/beta/asset-manager/transformations-focus.html#target-filename'>here</a>.
+        source:
+          type: object
+          required: ['attribute', 'channel', 'locale']
+          description: The attribute value in which is stored the media file you want to use as the source file for your transformation. More details <a href='/beta/asset-manager/transformations-focus.html#source-file'>here</a>.
+          properties:
+            attribute:
+              type: string
+            channel:
+              type: string
```

Comment:
Should you say "string or null" ? (same for others under)

Reply:
We can't with Swagger. I said it inside the plain doc.